hack/run-lint.sh

```diff
@@ -16,6 +16,10 @@
 
 set -euo pipefail
 
+[[ -n "${DEBUG:-}" ]] && set -x
+
+export GO111MODULE=on
```

Comment:
I personally prefer 

env a=b cmd args...

syntax as it restricts the scope of the variable and avoids other side effects. 
For example if we called install-gox.sh from one of these scripts, it would inherit this variable, and it would be potentially an issue.

Reply:
Fair point. I limited the scope to the essential parts.